test/test_playbooks/composite_content_view.yml

```diff
@@ -0,0 +1,58 @@
+---
+- hosts: fixtures
+  gather_facts: false
+  tasks:
+  - name: 'Load server config'
+    include_vars:
+      file: server_vars.yml
+  - include: tasks/organization.yml
+    vars:
+      organization_state: present
+  - include_tasks: tasks/product.yml
+    vars:
+      product_state: present
+  - include_tasks: tasks/repository.yml
+    vars:
+      product_state: present
+  - include_tasks: tasks/content_view.yml
+    vars:
+      content_view_state: present
+  - include_tasks: tasks/content_view_publish.yml
+  - include_tasks: tasks/content_view_version_promote.yml
+
+- hosts: tests
+  gather_facts: false
+  tasks:
+  - name: Load server config
+    include_vars:
+      file: server_vars.yml
+  - include_tasks: tasks/composite_content_view.yml
+    vars:
+      composite_content_view_state: present
+  - include_tasks: tasks/composite_content_view.yml
+    ignore_errors: yes
```

Comment:
`ignore_errors` in the test part seems wrong.
Anyway, the first test seems to have run without an error. Good work!